
 * Serving Flask app "main" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
/home/pi/rpi_temp/flask_app/rpi/rpi_setup.py:9: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
  GPIO.setup(led, GPIO.OUT)
Traceback (most recent call last):
  File "/home/pi/rpi_temp/flask_app/main.py", line 39, in <module>
    app.run(host=os.environ["DHT_IP"], port=os.environ["DHT_PORT"])
  File "/usr/lib/python3.9/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'DHT_IP'
